src/components/interactive/Chip.jsx

```diff
@@ -58,14 +56,14 @@ Chip.propTypes = {
    */
   onClose: PropTypes.func,
   /** Specify the size of the chip */
-  size: PropTypes.oneOf(['sm', 'md', 'lg']),
+  size: PropTypes.oneOf(['s', 'm', 'l']),
```

Comment:
I'd personally just would remove `l` from chips, as it's technically not a valid size right now from the design perspective (I can follow up with this change if you agree, if it's outside the scope of this PR)